docs/shared/tutorials/react-standalone.md

```diff
@@ -392,22 +479,133 @@ Nx allows you to separate this logic into "local libraries". The main benefits i
 - better scalability in CI by enabling independent test/lint/build commands for each library
 - better scalability in your teams by allowing different teams to work on separate libraries
 
-### Creating Local Libraries
+### Create a Local Library
 
-Let's assume our domain areas include `products`, `orders` and some more generic design system components, called `ui`. We can generate a new library for each of these areas using the React library generator:
+Let's assume our domain areas include `products`, `orders` and some more generic design system components, called `ui`. We can generate a new library for these areas using the React library generator:
 
 ```
 nx g @nx/react:library products --unitTestRunner=vitest --bundler=none --directory=modules/products
+```
+
+Note how we use the `--directory` flag to place the library into a subfolder. You can choose whatever folder structure you like to organize your libraries.
+
+Nx tries to set up your workspace to work with the modular library architecture, but depending on your existing configuration, you may need to tweak some settings. In this repo, you'll need to do a few things in order to prepare for future steps.
+
+#### Lint Settings
+
+Install the `@nx/eslint-plugin` package:
+
+```
+npx nx add @nx/eslint-plugin
+```
+
+We want the `lint` task for the root `react-app` project to only lint the files for that project, so we'll change the `lint` command in `package.json`:
+
+```json {% fileName="package.json" %}
+{
+  "scripts": {
+    "lint": "eslint src --ext ts,tsx --report-unused-disable-directives --max-warnings 0"
+  }
+}
+```
+
+Now we need to update the `.eslintrc.cjs` file to extend the `.eslintrc.base.json` file:
+
+```js {% fileName=".eslintrc.cjs" highlightLines=[8,10] %}
+module.exports = {
+  root: true,
+  env: { browser: true, es2020: true },
+  extends: [
+    'eslint:recommended',
+    'plugin:@typescript-eslint/recommended',
+    'plugin:react-hooks/recommended',
+    './.eslintrc.base.json',
+  ],
+  ignorePatterns: ['!**/*', 'dist', '.eslintrc.cjs'],
+  parser: '@typescript-eslint/parser',
+  plugins: ['react-refresh'],
+  rules: {
+    'react-refresh/only-export-components': [
+      'warn',
+      { allowConstantExport: true },
+    ],
+  },
+};
+```
+
+We'll also set `enforceBuildableLibDependency` to `false` in `.eslintrc.base.json`.
```

Comment:
This should be `"off"`not `false`. Also, is this needed since we're bringing back module boundaries later? Does it cause errors?

Reply:
This is not needed if we fix the vue plugin to correctly differentiate between apps and libraries

Reply:
I mistead, I thought it was turning off the lint rule. `false` is fine.